lib/sshkit/coordinator.rb

```diff
@@ -27,7 +27,7 @@ def each(options={}, &block)
     private
 
     def default_options
-      { in: SSHKit.config.default_runner }
+      { in: SSHKit.config.default_runner }.merge(SSHKit.config.default_runner_config)
```

Comment:
I was wondering, is it necessary to merge the `default_runner_config` here or could we just initialize `@default_runner_config` to `{ in: SSHKit.config.default_runner }` in sshkit/configuration.rb?